oslabs-beta/apollo11
# Graphiql endpoint detecting Apollo Client URI

Feature: Graphiql endpoint detecting Apollo Client URI
Description:
- Graphiql tab now detects apollo client URI and loads docs for endpoint
Changes I Made:
- App changed from class to functional component
- React useEffect added with hooks to receive message with apollo URI on App load
- Apollo URI passed to MainDrawer then to GraphiQL as props
- GraphiQL tab uses Apollo URI for query and docs
How to Test:
- Install the unpacked extension in the browser
- Navigate to a website using apollo
- Click on the graphiQL tab
- Refresh the website 
- GraphiQL tab should display the endpoint
- Instrospection query should also be sent to load docs
- GraphQL queries and results can be sent to the endpoint from within the tab
